docs/api/clients.rst

```diff
@@ -0,0 +1,93 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+Clients
+=======
+
+This section documents everything related to :class:`Client` and it's connectivity to Discord.
```

Comment:
adding something like, "if this is your first time here, check out the getting started pages"

Reply:
Done in [`d15ddd7`](https://github.com/DisnakeDev/disnake/pull/722/commits/d15ddd70e20c611070a3445379763a80b07a8c13)